tests/test_glacier/test_glacier_jobs.py

```diff
@@ -96,5 +84,7 @@ def test_get_job_output():
     })
     job_id = job_response['JobId']
 
+    time.sleep(62)
```

Comment:
@kefo Why do we need this? Is there a way to accomplish the same thing without adding a minute to all test runs?

Reply:
We could use freezegun to test this (though it would have to raise SkipTest when in server mode)
E.g. https://github.com/spulec/moto/blob/master/tests/test_sqs/test_sqs.py#L760-L796

Reply:
I wasn't crazy about it either; I'm open to ideas.

The pull request, in part, makes moto glacier work more like aws glacier, which is to say that when you now initiate a job to retrieve an archive, moto glacier does not immediately return the content.  Instead, a job is created (like aws), complete with an InProgress status.  You can query the job, etc.  After a minute (the 62 seconds you see in the test), it flips to a 'Completed' status and the client can fetch the archived content.

Reply:
What if we just used a shorter timeout? So keep the same functionality but change
```
self.et = self.st + datetime.timedelta(minutes=1)
```
to
```
self.et = self.st + datetime.timedelta(seconds=5)
```

Reply:
@kefo I have just one question about whether we can lower the sleep from 62 seconds to, say, 5 seconds. If we can, great. If not, that's fine too - either way we can merge this once we resolve that. Thanks!